sdcm/utils/common.py

```diff
@@ -1232,6 +1232,29 @@ def get_s3_scylla_repos_mapping(dist_type='centos', dist_version=None):
     return _S3_SCYLLA_REPOS_CACHE[(dist_type, dist_version)]
 
 
+ScyllaProduct = Literal['scylla', 'scylla-enterprise']
+
+
+def get_latest_scylla_release(region: str = 'eu-west-1',
```

Comment:
Don't we have a better way to get the latest release?
Somewhere that releng actively updating for every push?
(Although we need to avoid cases that they have problem with updating this as well, but we can warn also if it's older than X days).

Reply:
I've found out today we have this:
https://repositories.scylladb.com/scylla/check_version?system=scylla
https://repositories.scylladb.com/scylla/check_version?system=scylla-enterprise

I think I'll use this, and maybe add a check in the AMI test for the AMIs ?

@roydahan  if what's older ? the artifact under test ? what would be the criteria 7days ?